deliverydb/__init__.py

```diff
@@ -0,0 +1,4 @@
+from psycopg.rows import dict_row as psycopg_dict_row
+
+
+dict_row = psycopg_dict_row
```

Comment:
`dict_row` is imported as `psycopg_dict_row` but also made available as `dict_row`. But I cannot find a use for either outside this module.

If this is due to an existing use somewhere else, I'd rather handle this function-namechange there using `try ... except ImportError`

Either way, the same with a few characters less:
```suggestion
from psycopg.rows import dict_row


psycopg_dict_row = dict_row
```

Reply:
The idea is to keep the hard dependency on `psycopg` in the `deliverydb` package.

Reply:
A consumer of `deliverydb` can access the `psycopg_dict_row` without hard depending on `psycopg` on code level.

Reply:
The dependency remains inside the `deliverydb` package.